src/mon/HealthMonitor.h

```diff
@@ -45,9 +45,10 @@ class HealthMonitor : public QuorumService
   virtual bool service_dispatch(MonOpRequestRef op);
 
   virtual void start_epoch() {
+    epoch_t epoch = get_epoch();
     for (map<int,HealthService*>::iterator it = services.begin();
          it != services.end(); ++it) {
-      it->second->start(get_epoch());
```

Comment:
On Sun, Jul 17, 2016 at 06:02:17PM -0700, song.baisen wrote:
> >      for (map<int,HealthService*>::iterator it = services.begin();
> >           it != services.end(); ++it) {
> > -      it->second->start(get_epoch());
> 
> @jecluis  Hello.Thank you for review/How about change the tittle to small change  on the HealthMonitor start_epoch function .@branch-predictor  Hi! How do you think about this?

I agree with others, and I also doubt it'll be meaningful change -- unless 
you can prove that this change can save tangible amount of CPU processing 
time over reasonable mon lifetime. Otherwise it's more of code cleanup than
optimization. 

-- 
Piotr Dałek
branch@predictor.org.pl
http://blog.predictor.org.pl